CHANGES.md

```diff
@@ -1,3 +1,18 @@
+## edge-19.4.1
+
+* CLI
+  * Introduced an `upgrade` command! This allows an existing Linkerd Control Plane to be reinstalled or reconfigured
+  * The `inject` command proxy options are now converted into config annotations; the annotations ensure that these configs are persisted in subsequent resource updates
+  * Always show open TCP connections in the CLI
+  * The `stat` command now always shows the number of open TCP connections
+  * **Breaking Change** Removed the `--disable-external-profiles` flag from the `install` command; external profiles are now disabled by default and can be enabled with the new `--enable-external-profiles` flag
+* Controller
+  * The webhook config now reacts to changes in pods instead of deployments; this means pods parented not only by Depoloyments, but any kind of workload like Daemonsets and Statefulsets can be auto-injected.
+* Web UI
+  * The Octopus graph now shows TCP stats if no HTTP stats are available
```

Comment:
Might be better to call this the dependency graph or the topology graph, since I think 🐙 graph is an internal term